libs/tokens/src/const/tokensLists.ts

```diff
@@ -0,0 +1,4 @@
+import { ListsSourcesByNetwork } from '../types'
+import tokensList from './tokensList.json'
+
+export const DEFAULT_TOKENS_LISTS: ListsSourcesByNetwork = tokensList
```

Comment:
I wonder. If we save the JSON in a const, and then we add it to an atom...
- The memory from the JSON will never be liberated, right?
- Then we create the atom using the JSON, so its duplicated memory
- The constant is never used again. 

Would it make sense to load the JSON in an init function, so we don't need to keep a constant that is used only once?

Reply:
Got rid of it:
https://github.com/cowprotocol/cowswap/pull/3201/commits/b12a42962a6dfa7c3698e336143d45278381d0da
Thanks!